### PR TITLE
Update KBD67 readme so that it mentions the KBD65 PCB

### PR DESCRIPTION
**Pull request by Hackerpilot**
## Description

Document the fact that the KBD65 PCB is supported by this part of the firmware. It was not obvious that this was the case.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [X] Documentation

## Issues Fixed or Closed by This PR

None

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mechmerlin** [comment]
Are they two separate PCBs? Or is it actually one pcb called the KBD65?

**Comment by noroadsleft** [comment]
I think it's one PCB, but I can't figure out if KBDfans calls it KBD65 or KBD67.

If you search `kbd65` on their site, the only result you get in [this PCB](https://kbdfans.cn/products/kbd65-65-custom-mechanical-keyboard-pcb). The first image on that listing is [this](https://cdn.shopify.com/s/files/1/1473/3902/products/1_1800x1800.gif?v=1544176657). I had opened this image recently in Photoshop to try to measure the center nub distances, and got this:

image

Each of those six small red dots is about 63 pixels from the next; approximately 1u each.

If you then search `kbd67`, you find, among other things:

- [R2 [In stock]KBD67 Mechanical keyboard diy kit](https://kbdfans.cn/products/coming-soon-kbd67-mechanical-keyboard-diy-kit)
- [TOFU65 Custom mechanical keyboard DIY KIT](https://kbdfans.cn/products/tofu65-custom-mechanical-keyboard-diy-kit)
- [[in-stock]Tada68 mechanical keyboard 65% ISO layout](https://kbdfans.cn/products/tada68-mechanical-keyboard-65-iso-layout)

The first two of those listings make reference to a "KBD67" PCB, and the Tada68 references a "KBD65" PCB.

The KBD65 PCB listing and the KBD67 Keyboard Kit listing both reference [this image](https://cdn.shopify.com/s/files/1/1473/3902/files/12_f31494ef-5a3c-472e-b19c-79de733be78e.jpg?v=1540818282) for supported layouts. The Tada68 ISO listing says to use qmkeyboard.cn for programming, and to select "KBD67." Doing that results in a layout with 6 1u keys to the right of the spacebar (a layout that isn't actually supported by the KBD67 kit). So the PCB references use both "KBD65" and "KBD67", and the given programming link calls it "KBD67."

The Tofu kit listing gives the choice of opting for a Tada68 PCB that runs TMK, or a KBD67 PCB that runs QMK.

My personal opinion is until we have confirmation or proof that there are two different PCBs, we should note that the KBD67 firmware supports the KBD65, and support "both" that way.

**Comment by Hackerpilot** [comment]
> The KBD65 PCB listing and the KBD67 Keyboard Kit listing both reference [this image](https://cdn.shopify.com/s/files/1/1473/3902/files/12_f31494ef-5a3c-472e-b19c-79de733be78e.jpg?v=1540818282) for supported layouts.

I think that that image is incomplete. It depends on the case that you use.

I bought the following:
* https://kbdfans.cn/collections/keyboard-part/products/65-cnc-aluminum-plate
* https://kbdfans.cn/collections/65/products/kbd65-65-custom-mechanical-keyboard-pcb
* https://kbdfans.cn/collections/65-layout-case/products/in-stocktofu-65-aluminum-case

yolch65

The Tofu case doesn't have blockers, so you can have 6 1u keys on the right of the space.

The firmware installed properly with:

`make kbdfans/kbd67/rev1:hackerpilot:dfu`

**Comment by mechmerlin** [comment]
If it's one PCB, I would be inclined to change the name of this PCB to KBD65 and write in the notes that it is the PCB for the KBD67 keyboard. @Hackerpilot would you mind doing more digging into this, and make the changes?

**Comment by Hackerpilot** [comment]
Actually it may make sense to go the other way... I was just clearing off a table and found the packaging for the "KBD65" PCB. There's a sticker on it that says "kbd67". I think that this is mostly a problem with the KBDFans website not being up to date or consistent with itself.

**Comment by Hackerpilot** [comment]
So I should probably change the documentation to say "This PCB is sometimes sold under the name KBD65", though it would be better if KBDFans would just fix the product information on their site.

**Comment by noroadsleft** [comment]
@Hackerpilot 

> I think that that image is incomplete. It depends on the case that you use.

I failed to touch on that directly, but I agree with you that the image is incomplete. As you've said, the Tofu doesn't have a blocker, so you can use the full 6x 1u layout on the PCB.

@mechmerlin IMO it would be better, especially in light of @Hackerpilot's discovery that the packaging says "kbd67," to leave it under that name and note that it's sometimes sold/marketed as "KBD65."

**Comment by mechmerlin** [comment]
@Hackerpilot If the packaging says kbd67 who are we to argue with the manufacturer? lol. I think it's a good idea to make the readme more verbose. Please make the changes you recommended.

File: keyboards/kbdfans/kbd67/readme.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # KBD67
 
-A 65% keyboard sold in two variants. Rev1 was a typical keyboard that had to be soldered together, supporting multiple layouts. 
-Months later, a new version with hotswap was sold. **Firmware files from one, will not work on the other.** Please use the `.hex` appropriate for your board. 
+A 65% keyboard sold in two variants. Rev1 was a typical keyboard that had to be soldered together, supporting multiple layouts.
+Months later, a new version with hotswap was sold. **Firmware files from one, will not work on the other.** Please use the `.hex` appropriate for your board.
+The rev1 PCB for the KBD76 is sold under the name "KBD65"
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/readme.md]: Fixing the typo regarding the rev1, and making this line its own paragraph.

```suggestion

The rev1 PCB for the KBD67 is sold under the name "KBD65."
```

**Pull request merged**